modules/nw-egress-ips-config-object.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/assigning-egress-ips-ovn.adoc
+
+[id="nw-egress-ips-config-object_{context}"]
+= EgressIPconfig object
+As a feature of egress IP, the `reachabilityTotalTimeoutSeconds` configures the total timeout for checks that are sent by probes to egress IP nodes. The`egressIPConfig` object allows users to set the `reachabilityTotalTimeoutSeconds` `spec`. If the EgressIP node cannot be reached within this timeout, the node is declared down.
```

Comment:
s/`reachabilityTotalTimeoutSeconds`/`reachabilityTotalTimeoutSeconds` parameter (or whatever)

Reply:
s/The`egressIPConfig`/The `egressIPConfig`

Add a space; the marked up text isn't rendering correctly.